web/src/engine/predictive-text/worker-thread/src/main/correction/context-tracker.ts

```diff
@@ -127,15 +136,7 @@ export class TrackedContextState {
       // Be sure to deep-copy the tokens!  Pointer-aliasing is bad here.
```

Comment:
"pointers" are not a thing in JS. But objects are.

Reply:
The only difference between pointers and object references is the ability to directly apply arithmetic.  The 'aliasing' aspect is what's key here, though.